dotcom-rendering/src/lib/useMatchMedia.ts

```diff
@@ -7,6 +7,9 @@ import { useEffect, useState, useSyncExternalStore } from 'react';
  */
 const cachedQueries = new Map<string, MediaQueryList>();
 
+const removeMediaRulePrefix = (query: string): string =>
+	query.replace('@media', '').trim();
```

Comment:
In SDC this was done inside a [custom hook](https://github.com/guardian/support-dotcom-components/blob/main/packages/modules/src/hooks/useMediaQuery.ts#L11). But I think it's preferable to use the existing hook in this file (`useMatchMedia`) and call this new utility function where necessary (`removeMediaRulePrefix`)